skeletons/INTEGER.h

```diff
@@ -43,6 +43,8 @@ xer_type_encoder_f INTEGER_encode_xer;
 per_type_decoder_f INTEGER_decode_uper;
 per_type_encoder_f INTEGER_encode_uper;
 
+#define INTEGER_constraint asn_generic_no_constraint
```

Comment:
I know use `#define INTEGER_constraint asn_generic_no_constraint` might not be a good idea, but it looks like the original generated code works in this way.

I added this line because I found that the following definitions of `rfc3280-PKIX1Explicit88.asn1`  generate Version.c and CertificateSerialNumber.c and they use `INTEGER_constraint` without definition.

```
Version  ::=  INTEGER  {  v1(0), v2(1), v3(2)  }
CertificateSerialNumber  ::=  INTEGER
```

Then I check original generated Version.c and CertificateSerialNumber.c, there asn_constr_check_f is 

```
int
Version_constraint(asn_TYPE_descriptor_t *td, const void *sptr,
			asn_app_constraint_failed_f *ctfailcb, void *app_key) {
	/* Replace with underlying type checker */
	td->check_constraints = asn_DEF_INTEGER.check_constraints;
	return td->check_constraints(td, sptr, ctfailcb, app_key);
}
int
CertificateSerialNumber_constraint(asn_TYPE_descriptor_t *td, const void *sptr,
			asn_app_constraint_failed_f *ctfailcb, void *app_key) {
	/* Replace with underlying type checker */
	td->check_constraints = asn_DEF_INTEGER.check_constraints;
	return td->check_constraints(td, sptr, ctfailcb, app_key);
}
```
And in INTEGER.c, the asn_constr_check_f field of asn_DEF_INTEGER is indeed asn_generic_no_constraint.  

I did not change anything substantially but only simplified the logic of searching asn_constr_check_f field of asn_DEF_Version and asn_DEF_CertificateSerialNumber to eliminate `Version_constraint` and `CertificateSerialNumber_constraint'.

I think the question becomes "whether, at least for the two ASN.1 expressions, shall they generate corresponding constraint check functions for them or not ?" 

However, it is beyond my original purpose of this serial pull requests.